js/scripts.js

```diff
@@ -0,0 +1,21 @@
+// scripts.js
+
+var 	a,
+	h,
+	triangleArea = getTriangleArea(10,15),
+	triangle1Area = getTriangleArea(12, 15),
+	triangle2Area = getTriangleArea(1, 15);
+
+function getTriangleArea(a, h) {
+
+	if ((a <= 0) || (h <= 0)) {
+    		console.log('Nieprawid�owe dane');
```

Comment:
when you want to end your function earlier, just after checking your if statements, add `return;` inside your `if`.